olastic-core/src/main/java/com/hevelian/olastic/core/api/uri/queryoption/expression/member/impl/BaseMember.java

```diff
@@ -0,0 +1,75 @@
+package com.hevelian.olastic.core.api.uri.queryoption.expression.member.impl;
+
+import com.hevelian.olastic.core.api.uri.queryoption.expression.member.ExpressionMember;
+import org.apache.olingo.server.api.ODataApplicationException;
+
+import static com.hevelian.olastic.core.utils.ProcessorUtils.throwNotImplemented;
+
+/**
+ * Base common class for any expression member.
+ *
+ * @author Taras Kohut
+ */
+public abstract class BaseMember implements ExpressionMember {
+    @Override
+    public ExpressionMember any() throws ODataApplicationException {
+        return throwNotImplemented("Not implemented");
```

Comment:
There are too many duplication of "Not implemented". Maybe just create a method with no arguments that will add this message by default?